tests/dec/test_dec.py

```diff
@@ -621,11 +785,12 @@ def test_lark_file_model_list_consistency():
     with open(filename) as lark_file:
         lines = lark_file.readlines()
         for line in lines:
-            if "MODEL_NAME.2" in line:
+            if "MODEL_NAME :" in line:
                 break
         models = line.split(":")[1].strip(" ").strip("\n").split('"|"')
         models = [m.strip('"') for m in models]
 
+        assert set(models) == set(known_decay_models)
```

Comment:
Why isn't the next line enough?

Reply:
I thought it might be useful in interpreting the test output to know if a model has simply been forgotten or if the order is wrong.

Reply:
Oh, but if forgotten the list would not match. Same for the order. So that's already covered unless I'm really missing a use-case. If yes then can you give me an example with [1,2,3,4]?

It is not a big issue to have the extra line but nicer not to have redundant stuff.

Reply:
You're right that both cases would be covered with the last line. I added this line as the test was failing during debugging and I wasn't sure if I had only messed up the order. I agree that it's redundant and have removed it.